### DISCUSSION TITLE
How to create a PUBLIC s3 bucket

### DISCUSSION THREAD

**Discussion by hnq90** [seaweedfs/seaweedfs#4269] [Q&A]
I couldn't find any solution to create a public bucket for storing my website assets (fonts, images...) after creating some private buckets (for sensitive contents).
Because after enabling authentication, public access is always denied.

So I think I have to use the Filer API to upload public contents.

Please help 🥺

**Comment by hnq90** [comment]
There's a PR for it: https://github.com/seaweedfs/seaweedfs/pull/4090

**Reply by kmlebedev** [reply]
There https://github.com/seaweedfs/seaweedfs/wiki/Amazon-S3-API use can use anonymous identity
